bundle-workflow/.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E722
```

Comment:
This is usage of `except` catch-all, which we use to trap any kind of build errors from components.